frontend/public/components/monitoring.tsx

```diff
@@ -1954,8 +1962,12 @@ type MonitoringResourceIconProps = {
 type Group = {
   rules: PrometheusRule[];
   file: string;
-  inverval: number;
+  interval: number;
```

Comment:
Like `PrometheusRule`, `Group` represents the format returned by the Prometheus API (https://prometheus.io/docs/prometheus/latest/querying/api/#rules), so should only be changed to better match the Prometheus API.